[2025-03-01T02:11:50.175+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-01T02:11:50.194+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_postgres_1.insert_orders manual__2025-03-01T02:11:43.742736+00:00 [queued]>
[2025-03-01T02:11:50.202+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_postgres_1.insert_orders manual__2025-03-01T02:11:43.742736+00:00 [queued]>
[2025-03-01T02:11:50.203+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-01T02:11:50.217+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): insert_orders> on 2025-03-01 02:11:43.742736+00:00
[2025-03-01T02:11:50.227+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=59852) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-01T02:11:50.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_postgres_1', 'insert_orders', 'manual__2025-03-01T02:11:43.742736+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/dag_generate_table_info.py', '--cfg-path', '/tmp/tmpxxu4ujw_']
[2025-03-01T02:11:50.228+0000] {standard_task_runner.py:72} INFO - Started process 59868 to run task
[2025-03-01T02:11:50.228+0000] {standard_task_runner.py:105} INFO - Job 96: Subtask insert_orders
[2025-03-01T02:11:50.268+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_postgres_1.insert_orders manual__2025-03-01T02:11:43.742736+00:00 [running]> on host e6b43e607126
[2025-03-01T02:11:50.344+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kucikm23@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_postgres_1' AIRFLOW_CTX_TASK_ID='insert_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T02:11:43.742736+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T02:11:43.742736+00:00'
[2025-03-01T02:11:50.346+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-01T02:11:50.346+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-01T02:11:50.346+0000] {logging_mixin.py:190} INFO - Current task name:insert_orders state:running start_date:2025-03-01 02:11:50.194949+00:00
[2025-03-01T02:11:50.347+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 and current dag run status:running
[2025-03-01T02:11:50.347+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-01T02:11:50.549+0000] {logging_mixin.py:190} INFO - order_id          0
customer_email    0
order_date        0
amount            0
currency          0
dtype: int64
[2025-03-01T02:11:50.558+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-03-01T02:11:50.616+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-01T02:11:50.626+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-01T02:11:50.627+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_postgres_1, task_id=insert_orders, run_id=manual__2025-03-01T02:11:43.742736+00:00, execution_date=20250301T021143, start_date=20250301T021150, end_date=20250301T021150
[2025-03-01T02:11:50.651+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-01T02:11:50.651+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-01T02:11:50.652+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 queued_at:2025-03-01 02:11:43.750862+00:00
[2025-03-01T02:11:50.652+0000] {logging_mixin.py:190} INFO - Task hostname:e6b43e607126 operator:PythonOperator
[2025-03-01T02:11:50.683+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-01T02:11:50.698+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T02:11:50.702+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
